app/apis/api_admin.py

```diff
@@ -0,0 +1,139 @@
+from functools import wraps
+from flask_jwt_simple import get_jwt_identity
+from flask_restplus import Resource, Namespace, fields, reqparse
+
+from auth.exceptions import ApplicationUserRoleException
+from models import db
+from models.application_user_role import ApplicationUserRole, Role
+from models.user import User
+
+admin_info_namespace = Namespace('admin', description='Admin Endpoint.')
+success_or_not = admin_info_namespace.model('Success', {
+    'status': fields.Boolean(
+        required=True
+    ),
+    'message': fields.String(
+        required=True
+    )
+})
+user_info = admin_info_namespace.model('User', {
+    'user_uid': fields.String(required=True),
+    'user_name': fields.String(required=True)
+})
+role_info = admin_info_namespace.model('Role', {
+    'role': fields.String(required=True),
+    'user': fields.Nested(user_info, required=True)
+})
+
+
+@admin_info_namespace.route('/users')
+class ApiApplicationUsers(Resource):
+    @admin_info_namespace.marshal_list_with(user_info)
+    def get(self):
+        return User.query.all()
+
+
+def check_role(fn):
```

Comment:
Naming `check_role` is duplicated [here](https://github.com/rekcurd/drucker-dashboard/pull/49/files#diff-3b16d374ef08532ca6b01246c5f0fa41R23) and I believe this method checks "owner" role.
How about using `check_owner_role` instead?

Reply:
fixed.